src/app/CommandHandler.h

```diff
@@ -162,6 +162,11 @@ class CommandHandler
     TLV::TLVWriter * GetCommandDataIBTLVWriter();
     FabricIndex GetAccessingFabricIndex() const;
 
+    /**
+     * Rollback the state to before encoding the current ResponseData (before calling PrepareCommand / PrepareStatus)
+     */
+    CHIP_ERROR RollbackResponse();
```

Comment:
Should this really be a public API?  When would this be used in practice as a public API?

Reply:
This should be a private API.